apps/console/src/features/applications/components/settings/info.tsx

```diff
@@ -0,0 +1,114 @@
+    OIDCApplicationConfigurationInterface,
+    SAMLApplicationConfigurationInterface
+} from "../../models";
+import { OIDCConfigurations, SAMLConfigurations } from "../help-panel";
+
+/**
+ * Proptypes for the server endpoints details component.
+ */
+interface InfoPropsInterface extends LoadableComponentInterface, TestableComponentInterface {
+    inboundProtocols: InboundProtocolListItemInterface[];
+    isSAMLConfigLoading: boolean;
+    isOIDCConfigLoading: boolean;
+}
+export const Info: FunctionComponent<InfoPropsInterface> = (
+    props: InfoPropsInterface
+): ReactElement => {
+
+    const { inboundProtocols, isOIDCConfigLoading, isSAMLConfigLoading  } = props;
+    const oidcConfigurations: OIDCApplicationConfigurationInterface = useSelector(
+        (state: AppState) => state.application.oidcConfigurations);
+    const samlConfigurations: SAMLApplicationConfigurationInterface = useSelector(
+        (state: AppState) => state.application.samlConfigurations);
+    const { t } = useTranslation();
+    const [isOIDC, setIsOIDC] = useState<boolean>(false);
+    const [isSAML, setIsSAML] = useState<boolean>(false);
+    const [ isLoading, setIsLoading ] = useState<boolean>(false);
+
+    useEffect(() => {
+        if (inboundProtocols == undefined) {
+            return;
+        }
+
+        if (inboundProtocols.length > 0) {
+            inboundProtocols.map((protocol) => {
+                if (protocol.type == "oauth2") {
+                    setIsOIDC(true);
+                    setIsLoading(isOIDCConfigLoading);
+                } else if (protocol.type == "samlsso") {
+                    setIsSAML(true);
+                    setIsLoading(isSAMLConfigLoading);
+                }
+            });
+        }
+    }, [inboundProtocols]);
+
+    return (
+        !isLoading ? (
+            <EmphasizedSegment loading={ isLoading } padded="very">
+                <Grid className="form-container with-max-width">
+                    <Grid.Row>
+                        <Grid.Column>
+                            <Heading ellipsis as="h4">
+                                <strong>
+                                    { t("console:develop.features.applications.helpPanel.tabs.start.content." +
+                                        "endpoints." + "title") }
+                                </strong>
+                            </Heading>
+
+                            { t("console:develop.features.applications.helpPanel.tabs.start.content.endpoints." +
+                                            "subTitle") }
+
+                            <Divider hidden/>
+                            { isOIDC && (
+                                <OIDCConfigurations oidcConfigurations={ oidcConfigurations }/>
+                            ) }
+                            { isOIDC && isSAML ? (
+                                <>
+                                    <Divider hidden/>
+                                    <Divider/>
```

Comment:
Try adding a className of `x2` or `x3`. And remove the two hidden dividers.
```suggestion
                                    <Divider className="x2"/>
```